packages/pigeon/bin/run_tests.dart

```diff
@@ -87,53 +87,41 @@ Future<int> _runDartUnitTests() async {
   return exitCode;
 }
 
+Future<int> _generateDart(Map<String, String> jobs) async {
```

Comment:
I still had to read the implementation to know what this meant, since without much context "input" and "output" are extremely generic terms.

How about: "Generates multiple dart files via `pigeon` invocations based on the jobs [...] the format of (key: pigeon input file, value: pigeon dart output file)."